src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.OrderDto;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/order")
+public class OrderController {
```

Comment:
pozostałe kontrolery nie używają prefixu /api/ 
to doprowadzi do wielu pomyłek

Reply:
poprawione